src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -322,6 +322,20 @@ class ErgoWalletActor(settings: ErgoSettings,
       walletVars = walletVars.resetProver()
       secretStorageOpt.foreach(_.lock())
 
+    case RescanWallet =>
+      val registryFolder = WalletRegistry.registryFolder(settings)
+
+      log.info(s"Rescanning the wallet, its registry is in $registryFolder")
+      val rescanResult = Try {
+        registry.close()
+        FileUtils.deleteRecursive(registryFolder)
+        registry = WalletRegistry.apply(settings)
+      }
+      rescanResult.recover { case t =>
+        log.error("Error during rescan attempt: ", t)
+      }
+      sender() ! rescanResult
```

Comment:
I see the registry is recreated, but when the scan actually happens? Please add the clarification in the code.

Reply:
Frankly speaking I do not know answer for this question

Reply:
@knizhnik @aslesarenko added resync signal sending and comments